common-util/Details/ServiceState/utils.jsx

```diff
@@ -266,3 +426,80 @@ export const onStep5Unbond = (account, id) => new Promise((resolve, reject) => {
       reject(e);
     });
 });
+
+/* ----- operator whitelist functions ----- */
+export const checkIfServiceRequiresWhiltelisting = (serviceId) => new Promise((resolve, reject) => {
+  const contract = getOperatorWhitelistContract();
+
+  contract.methods
+    .mapServiceIdOperatorsCheck(serviceId)
+    .call()
+    .then((response) => {
+      // if true: it is whitelisted by default
+      // else we can whitelist using the input field
+      resolve(response);
+    })
+    .catch((e) => {
+      reject(e);
+      notifyError('Error occured on checking if service requires whitelisting');
+    });
+});
+
+export const checkIfServiceIsWhitelisted = (serviceId, operatorAddress) => new Promise((resolve, reject) => {
+  const contract = getOperatorWhitelistContract();
+
+  contract.methods
```

Comment:
Not for this PR but it feels like there's a lot of duplicate code in these contract calls